sdk/dotnet/Rds/Inputs/ParameterGroupParameterArgs.cs

```diff
@@ -34,6 +34,7 @@ public sealed class ParameterGroupParameterArgs : global::Pulumi.ResourceArgs
 
         public ParameterGroupParameterArgs()
         {
+            ApplyMethod = "immediate";
```

Comment:
I wonder why we didn't generate the defaults in the sdk before - this PR just moves the defaults from the TF schema to the pulumi overlay. IMO this is a better UX for the sdks, does anyone know why they didn't show up before?

Reply:
I believe it depends on how the defaults are applied. We can't always see inside TF providers to find the defaults they supply.